app/assets/css/sass/_rtm.scss

```diff
@@ -547,3 +547,24 @@ input.imgedit-submit-btn {
 .hide_privacy_wrapper {
 	display: none;
 }
+
+.avatar {
+	display: block;
+	height: auto;
+	min-height: inherit;
+	width: auto;
+}
+
+h2:before {
+	height: 0;
+	margin: 0;
+	width: 0;
+}
+
+#buddypress div#item-header-cover-image .user-nicename {
+	margin: 0 0 5px;
+}
+
+#buddypress #item-header-content {
+	padding-left: 37px;
+}
```

Comment:
Add new line at end

Reply:
You are checking on old commit. Please check my post commit